packages/core/strapi/lib/core/registries/content-types.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { pickBy, has } = require('lodash/fp');
+const { createContentType } = require('../domain/content-type');
+
+// const validateContentTypesUnicity = contentTypes => {
```

Comment:
Should we finish this before merging ?

Reply:
I put a todo. I would prefer to handle it a bit later if it's ok for you